doc/changes.rst

```diff
@@ -1,6 +1,110 @@
+* Add a gallery example for logo (#823)
+* Update axis label explanation in frames tutorial (#820)
+* Update GMT arguments in coast doc strings (#798)
+* Add guidelines for types of tests to write (#796)
+* Recommend using SI units in documentation (#795)
+* Create gallery example for datetime inputs (#779)
+* Add an example for plotting multi-parameter symbols (#772)
+* Add a gallery example for the fig.image() method (#777)
+* Add a gallery example of setting line colors with a custom CPT (#774)
+* Add a table for compatibility of PyGMT with Python and GMT (#763)
+* Add description for the "columns" arguments (#766)
+* Add table for projections (#753)
+* Add more gallery examples for projections (#761, #721, #757, #723, #762, #742)
+* Add projection description for Lambert Azimuthal Equal-Area (#760)
+* Change text when GMTInvalidInput error is raised for basemap (#729)
+* Add non-geographic plots to projection gallery (#728)
+* Adding cylindrical projections to gallery (#727)
+* Add tutorials, for adding a map title (#720), plotting Earth relief (#712), 3D perspective image (#743), contour maps (#705), plotting lines tutorial (#741), region arguments (#800)
+* Add frame and cmap to the colorbar parameters list (#709)
+* Add frame as a parameter on the grdview documentation (#707)
+
+Bug Fixes
+
+* Fix a bug of Figure.text() when "text" is a non-string array (#724)
+* Fix the error message when IPython is not available (#701)
+
+Maintenance
+
+* Add dependabot to keep GitHub Actions up to date (#861)
+* Add authorship policy (#726)
+* Move plotting functions to separate modules (#808)
+* Skip workflows in PRs if only non-source-code files are changed (#839)
+* Move non-plotting modules to separate files in src directory (#832)
+* Show test execution times in pytest (#835)
+* Add slash command '/test-gmt-dev' to test GMT dev version (#831)
+* Add tests for grdfilter (#809)
+* Improve the error message when PyGMT fails to load the GMT library (#814)
+* Add tests for GMTInvalidInput for figure.py (#810)
+* Split meca into a standalone module from base_plotting.py (#686)
+* Use args_in_kwargs for basemap error raising (#797)
+* Add docformatter to format plain text in docstrings (#642)
+* Add GMTInvalidInput Error for Figure.coast (#787)
+* Add args_in_kwargs function (#791)
+* Add a CI job testing GMT master branch on Windows (#756)
+* Add isort to sort imports alphabetically (#745)
+* Convert relative imports to absolute imports (#754)
+* Add a Makefile target 'distclean' for deleting project metadata files (#744)
+* Add test test_basemap_map_scale (#739)
+* Check files for UNIX-style line breaks and 644 permission (#736)
+* Rename vercel configuration file from now.json to vercel.json (#738)
+* Migrate pytest configurations to pyproject.toml (#725)
+* Migrate documentation deployment from Travis CI to GitHub Actions (#713)
+* Switch from versioneer to setuptools-scm (#695)
+* Migrate coverage configurations to pyproject.toml (#667)
```

Comment:
Roughly group the changes into categories (CI, source codes, documentation, tests, and misc), and update description of a few changes. The changes in the 'misc' category (the last three entries) may be moved into "Enhancements" instead.

**NOTE**: Need to remove the blank lines before committing the changes.

```suggestion
* Add dependabot to keep GitHub Actions up to date (#861)
* Skip workflows in PRs if only non-source-code files are changed (#839)
* Add slash command '/test-gmt-dev' to test GMT dev version (#831)
* Check files for UNIX-style line breaks and 644 permission (#736)
* Rename vercel configuration file from now.json to vercel.json (#738)
* Add a CI job testing GMT master branch on Windows (#756)
* Migrate documentation deployment from Travis CI to GitHub Actions (#713)

* Move Figure.meca() into a standalone module (#686)
* Move plotting functions to separate modules (#808)
* Move non-plotting modules to separate modules (#832)
* Add isort to sort imports alphabetically (#745)
* Convert relative imports to absolute imports (#754)
* Switch from versioneer to setuptools-scm (#695)

* Add authorship policy (#726)
* Add docformatter to format plain text in docstrings (#642)

* Migrate pytest configurations to pyproject.toml (#725)
* Migrate coverage configurations to pyproject.toml (#667)
* Show test execution times in pytest (#835)
* Add tests for grdfilter (#809)
* Add tests for GMTInvalidInput of Figure.savefig() and Figure.show() (#810)
* Add args_in_kwargs function (#791)
* Add a Makefile target 'distclean' for deleting project metadata files (#744)
* Add a test Figure.basemap() map_scale (#739)

* Improve the error message when PyGMT fails to load the GMT library (#814)
* Add GMTInvalidInput Error for Figure.coast (#787)
* Use args_in_kwargs for basemap error raising (#797)
```

Reply:
I think the maintenance/documentation PR 726 could probably fit under the actual documentation section, but keep 642 in maintenance. Agree that 814 and 787 could be under enhancements, but maybe not 787 (keep it under maintenance) since that is non-user facing.

Feel free to make the change, I'm gonna have dinner first.